### ISSUE TITLE
--fix option doesnt work 

### ISSUE THREAD

**Issue by netei** [mantoni/eslint_d.js#19]
Take this `.eslintrc`

```
{
  "rules": {
    "space-infix-ops": 2
  }
}
```

and this `js`

```
var foo=function (bar) {
  return bar;
};
```

`eslint .`  and `eslint_d .` both produce the same output : 

```
/tmp/eslint_t/bootstrap.js
  1:8  error  Infix operators must be spaced  space-infix-ops
```

However 

`eslint . --fix` does modify the file 
`eslint_d . --fix` doesn't modify the file

**Comment by netei** [comment]
```
eslint_d --version
v1.9.0 (eslint_d v2.3.2)
```

**Comment by zurawiki** [comment]
Can confirm `--fix` isn't working for me either. I would like to see this bug fixed

**Comment by mantoni** [comment]
Sorry to let you wait this long. Fixed in `2.5.1`.

**Comment by netei** [comment]
Great :-)
